src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -185,7 +199,14 @@ public static void performMinimization(
         parsedTargetFiles.put(targetFile, parseJavaFile(root, targetFile));
       }
       for (String targetFile : addMissingClass.getAddedTargetFiles()) {
-        parsedTargetFiles.put(targetFile, parseJavaFile(root, targetFile));
+        try {
+          parsedTargetFiles.put(targetFile, parseJavaFile(root, targetFile));
+        } catch (ParseProblemException e) {
+          // These parsing codes cause crashes in the CI. Those crashes can't be reproduced locally.
+          // Not sure if something is wrong with VineFlower or Specimin CI. Hence we keep these
+          // lines as tech debt.
+          continue;
```

Comment:
This does make me very nervous, but I agree it's okay to keep it for now. Please make this comment a TODO, and add another TODO at each of the other places that you've added such a catch below.